packages/validator/src/util/externalSignerClient.ts

```diff
@@ -104,13 +105,13 @@ type Web3SignerSerializedRequest = {
 /**
  * Return public keys from the server.
  */
-export async function externalSignerGetKeys(externalSignerUrl: string): Promise<string[]> {
+export async function externalSignerGetKeys(externalSignerUrl: string, logger?: Logger): Promise<string[]> {
   const res = await fetch(`${externalSignerUrl}/api/v1/eth2/publicKeys`, {
     method: "GET",
     headers: {"Content-Type": "application/json"},
   });
 
-  return handlerExternalSignerResponse<string[]>(res);
+  return handlerExternalSignerGetKeysResponse(res, logger);
```

Comment:
since this handler is now specific to keys request, having the request handling in a separate function is not required (same for other functions)

Reply:
fixed